### DISCUSSION TITLE
功能优化和建议 - Function optimization and suggestions

### DISCUSSION THREAD

**Discussion by baikaishuipp** [baikaishuipp/jcci#4] [Ideas]
对于JCCI有任何功能上的建议都可以参与讨论，有价值的建议会加入后续的迭代更新中，谢谢~

If you have any functional suggestions for JCCI, you can participate in the discussion. Valuable suggestions will be added to subsequent iterative updates. Thank you~

**Comment by Leofighting** [comment]
可以在生成的结果文件中，把对应函数、类的注释内容也一并带过来，方便测试人员理解

**Reply by baikaishuipp** [reply]
好的，会考虑的

**Comment by songliqiang520** [comment]
建议生成的报告：列表形式或者思维导图的形式，现在如果改动太大，屏幕小的话，看不全

**Reply by baikaishuipp** [reply]
好的，有精力再做
